weatherAPI/libraries/response.go

```diff
@@ -0,0 +1,152 @@
+package libraries
+
+import (
+	"encoding/json"
+	"fmt"
+	"time"
+
+	"github.com/astaxie/beego/httplib"
+	"github.com/astaxie/beego/logs"
+)
+
+func GetResponse(city, country string) map[string]interface{} {
+	var jsonResponse map[string]interface{}
+	response := make(map[string]interface{})
+
+	uri := fmt.Sprintf("http://api.openweathermap.org/data/2.5/weather?q=%v,%v&appid=1508a9a4840a5574c822d70ca2132032", city, country)
+	res := httplib.Get(uri) //get the uri sending
+	res.Debug(true)
+	str, e := res.String() //return raw response body
+	CheckErrors("Error returning the raw response: ", e)
+	err := json.Unmarshal([]byte(str), &jsonResponse)
```

Comment:
Please, use a defined struct to unmarshall all the responses from the external API.